ush/forecast_predet.sh

```diff
@@ -568,7 +568,13 @@ CICE_predet(){
 
   # CICE does not have a concept of high frequency output like FV3
   # Convert output settings into an explicit list for CICE
-  CICE_OUTPUT_FH=$(seq -s ' ' "${FHMIN}" "${FHOUT_OCNICE}" "${FHMAX}")
+  if (( $(( ( cyc + FHMIN ) % FHOUT_ICE )) == 0 )); then
+    mapfile -t -d ' ' CICE_OUTPUT_FH < <(seq -s ' ' "${FHMIN}" "${FHOUT_ICE}" "${FHMAX}") || true
```

Comment:
I don't think these should pass if the list generation fails
```suggestion
    mapfile -t -d ' ' CICE_OUTPUT_FH < <(seq -s ' ' "${FHMIN}" "${FHOUT_ICE}" "${FHMAX}") || exit 10
```

Reply:
@WalterKolczynski-NOAA It looks like this triggered a spellcheck error. Is it ok to do a spellcheck disable for this (`# shellcheck disable=SC2312`)? It looks like this particular spellcheck was disabled in instances where mapfile was used in line 102 and line 544 in `forecast_predet`.

Reply:
Yes, go ahead and disable